FabNeso.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+#
+# This source file is part of the FabSim software toolkit, which is
+# distributed under the BSD 3-Clause license.
+# Please refer to LICENSE for detailed information regarding the licensing.
+#
+# This file contains FabSim definitions specific to FabNeso
+#
+# authors: Duncan Leggat
```

Comment:
```suggestion
"""
Task definitions for FabNESO plug-in to FabSIM software toolkit.

Defines tasks for running simulations using Neptune Exploratory Software (NESO).
"""
```

Typical convention is to document what a module does using a multiline string (`""" ... """`) which will get assigned to `__doc__` attribute (and used in autogenerated documentation). The LICENSE details I think are redundant as we already have the license specified in root of repository (and we've currently got MIT license here). Similar `authors` is likely to get out of date quickly - better to record this in metadata in `pyproject.toml`.

Reply:
The header comment was written automatically when making the plugin and is consistent with the other, existing plugins. I'm happy to change it in the way you describe, just noting that this is another one of those 'default FabSIM' things!

Reply:
Yeah sorry realised this is from FabSIM side rather than something you've added. It looks like in general their code is a bit out of date with current Python standards. While there is some value in keeping with the existing conventions in a project, in this case I think there are enough downsides (using an inline comment will mean no `__doc__` attribute is set, having an author list and licence details in comment makes it difficult to ensure these are kept up to date) that I would err with breaking their convention.

Reply:
I've gone ahead and updated these